conf/nginx/sites-available/obf

```diff
@@ -66,17 +69,19 @@ server {
 
 	location /data/ {
 		include snippets/off.cors-headers.include;
+		include snippets/exiry-headers.include;
```

Comment:
```suggestion
		include snippets/expiry-headers.include;
```

The typo is repeated 20 times. The actual file is named correctly.

Reply:
:grimacing: thanks !

Reply:
fixed.